common/src/verify.rs

```diff
@@ -114,33 +114,40 @@ enum Phase {
     Block,
 }
 
-/// Verifies whether the given sequence of commits can be agenda subset of agenda finalized chain.
+/// Verifies whether the given sequence of next_block_commits can be agenda subset of agenda finalized chain.
 ///
 /// It may accept sequences that contain more than one `BlockHeader`.
 #[derive(Debug, Clone)]
 pub struct CommitSequenceVerifier {
     header: BlockHeader,
     phase: Phase,
     reserved_state: ReservedState,
-    commits: Vec<Commit>,
+    next_block_commits: Vec<Commit>,
+    total_commits: Vec<Commit>,
 }
 
 impl CommitSequenceVerifier {
     /// Creates agenda new `CommitSequenceVerifier` with the given block header.
     pub fn new(start_header: BlockHeader, reserved_state: ReservedState) -> Result<Self, Error> {
         Ok(Self {
-            header: start_header,
+            header: start_header.clone(),
             phase: Phase::Block,
             reserved_state,
-            commits: vec![],
+            next_block_commits: Vec::new(),
+            total_commits: vec![Commit::Block(start_header)],
         })
     }
 
+    /// Returns the next_block_commits received so far.
+    pub fn get_commits(&self) -> &[Commit] {
+        &self.total_commits
```

Comment:
```suggestion
        &self.next_block_commits
```

Reply:
No, this is right and the comment is wrong.

Reply:
fixed.